tests/gui/pages/main.spec.js

```diff
@@ -147,6 +150,33 @@ describe('Browser: MainPage', function() {
       m.chai.expect(_.sortBy(extensions)).to.deep.equal(_.sortBy(SupportedFormatsModel.getAllExtensions()));
     });
 
+    describe('.getImageBasename()', function() {
+
+      it('should return the basename of the selected image', function() {
+        const controller = $controller('ImageSelectionController', {
+          $scope: {}
+        });
+
+        SelectionStateModel.setImage({
+          path: path.join(__dirname, 'foo', 'bar.img'),
+          size: 999999999
+        });
```

Comment:
True. Maybe we should add some "factory" objects to create common things like images, drives, etc for testing purposes. What do you think?